src/main/java/HtmlUtil.java

```diff
@@ -1,50 +1,40 @@
 import fitnesse.responders.run.SuiteResponder;
 import fitnesse.wiki.*;
 
-
 public class HtmlUtil {
+	private static String SETUP_NAME = "SetUp";
+	private static String TEARDOWN_NAME = "TearDown";
 
     public static String testableHtml(PageData pageData, boolean includeSuiteSetup) throws Exception {
-        WikiPage wikiPage = pageData.getWikiPage();
-        StringBuffer buffer = new StringBuffer();
+		WikiPage wikiPage = pageData.getWikiPage();
+		StringBuffer buffer = new StringBuffer();
 
-        if (pageData.hasAttribute("Test")) {
-            if (includeSuiteSetup) {
-                WikiPage suiteSetup = PageCrawlerImpl.getInheritedPage(SuiteResponder.SUITE_SETUP_NAME, wikiPage);
-                if (suiteSetup != null) {
-                    WikiPagePath pagePath = wikiPage.getPageCrawler().getFullPath(suiteSetup);
-                    String pagePathName = PathParser.render(pagePath);
-                    buffer.append("!include -setup .").append(pagePathName).append("\n");
-                }
-            }
-            WikiPage setup = PageCrawlerImpl.getInheritedPage("SetUp", wikiPage);
-            if (setup != null) {
-                WikiPagePath setupPath = wikiPage.getPageCrawler().getFullPath(setup);
-                String setupPathName = PathParser.render(setupPath);
-                buffer.append("!include -setup .").append(setupPathName).append("\n");
-            }
-        }
+		if (pageData.hasAttribute("Test")) {
+			if (includeSuiteSetup) {
```

Comment:
Could be outsourced into function and "!include -setup ." could be more abstracted (pass e.g. "setup" as parameter)